extensions/wikia/SpecialDiscussionsLog/SpecialDiscussionsLogController.class.php

```diff
@@ -1,329 +1,15 @@
-		return $records;
-	}
-
-	private function getSearchResults( $url, $query ) {
-		$client = new \GuzzleHttp\Client();
-
-		try {
-			$response = $client->post(
-					$url, [
-							'body' => $query,
-					]
-			);
-		} catch ( \GuzzleHttp\Exception\RequestException $requestException ) {
-			$this->logger->error(
-					sprintf( 'Request to elasticsearch failed: %s', $requestException->getMessage() ),
-					[ 'exception' => $requestException ]
-			);
-			return false;
-		}
-
-		if ( $response->getStatusCode() !== self::HTTP_STATUS_OK ) {
-			$this->logger->error(
-					sprintf( 'Elasticsearch request error; status code %d', $response->getStatusCode() )
-			);
-			return false;
-		}
-
-		return $response;
-	}
-
-	private function appendRecordsFromResponse( array &$records, $response, array &$uniqueRecordChecker  ) {
-
-		$resultObject = json_decode( $response->getBody() );
-		$hits = $resultObject->hits->hits;
-
-		foreach ( $hits as $hit ) {
-			$record = $hit->_source;
-			$ip = $record-> { 'fastly_client_ip' } ;
-			$site = $record-> { 'site_id' } ;
-			$userId = $record-> { 'user_id' } ;
-			$userAction = isset( $record-> { 'user_action' } ) ? $record->{ 'user_action' } : null;
-			$recordHash = sprintf( '%s:%s:%s:%s', $ip, $site, $userId, $userAction );
-
-			if ( !empty( $uniqueRecordChecker[$recordHash] ) ) {
-				continue;
-			}
-
-			$uniqueRecordChecker[$recordHash] = true;
-
-			try {
-				$user = $this->getUserById( $userId );
-			} catch ( \Exception $e ) {
-				$this->logger->error(
-						sprintf( 'User not found: %s', $e->getMessage() ),
-						[ 'exception' => $e ]
-				);
-				continue;
-			}
-
-			if ( !isset( $domainCache[$site] ) ) {
-				$cityTitle = WikiFactory::getWikiByID( $site )->city_title;
-				$wikiDomain = preg_replace( '#^https?://#', '', WikiFactory::getHostById( $site ) );
-				if ( empty( $cityTitle ) && empty( $wikiDomain ) ) {
-					// most likely from a different environment
-					// (e.g. prod contribution when this is on dev, or vice versa)
-					$this->logger->warning( sprintf( 'Site not found: %d', $site ) );
-					continue;
-				} else {
-					$domainCache[$site] = ['title' => $cityTitle, 'domain' => $wikiDomain ];
-				}
-			}
-
-
-			$timestamp = strtotime( $record-> { '@timestamp' } );
-
-			$userLogRecord = new UserLogRecord();
-			$userLogRecord->site = $domainCache[$site];
-			$userLogRecord->ip = $ip;
-			$userLogRecord->timestamp = date( DATE_RFC2822, $timestamp );
-			$userLogRecord->userAgent = $record-> { 'user_agent' } ;
-			$userLogRecord->user = $user;
-			$userLogRecord->userAction = $userAction;
-
-			$records[$timestamp] = $userLogRecord;
-		}
-	}
-
-	private function checkAccess() {
-		if ( !$this->wg->User->isAllowed( self::DISCUSSIONS_LOG_ACTION ) ) {
-			throw new \PermissionsError( self::DISCUSSIONS_LOG_ACTION );
-		}
+		$urlHost = parse_url( $this->wg->title->getFullURL(), PHP_URL_HOST );
+		$this->wg->Out->redirect( 'https://' . $urlHost . '/d/log' );
```

Comment:
We don't support https yet. Also, you can use `$this->wg->Server` instead of parsing title url.